plugins/lsp/language-servers/efmls-configs-pkgs.nix

```diff
@@ -94,7 +95,6 @@ pkgs: {
       vulture
       ;
     inherit (nodePackages)
-      eslint
```

Comment:
this needs a fallback, too. Should be something like `eslint = pkgs.eslint or pkgs.nodePackages.eslint;`